.github/workflows/main.yml

```diff
@@ -125,7 +125,7 @@ jobs:
 
   macOS:
     name: macOS
-    runs-on: macos-latest
+    runs-on: macos-12
```

Comment:
So according to what I read this should be the same. Although I'm fine pinning this to avoid breakage.

Reply:
Yep, but before I changed it, it was failing :thinking:. So I don't know if something else changed during my tests, or if that is different, but I'm a bit confused.